users_ldap_mail/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': "LDAP mapping for user name and e-mail",
-    'version': "10.0.1.0.0",
+    'version': "11.0.1.0.0",
     'depends': ["auth_ldap"],
     'author': "Daniel Reis (https://launchpad.com/~dreis-pt),"
```

Comment:
@dreispt - are you good with removing your launchpad from this?

Reply:
Yes, of course 😄